acceptance/bundle/python/mutator-ordering/output.txt

```diff
@@ -0,0 +1,66 @@
+
+>>> uv run --python 3.10 --with databricks-bundles==0.7.2 -q [CLI] bundle validate --output json
```

Comment:
Tip: use `jq` to focus on the parts you care about in the test; here `experimental` and `resources.jobs`.

Reply:
👍 , fixed in script file